include/mantella_bits/numericalAnalysis.hpp

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+// C++ standard library
+#include <functional>
+
+namespace mant {
+
+  double brent(
+      std::function<double(double)> objectiveFunction,
+      double lowerBound,
+      double upperBound,
+      double xTolerance);
```

Comment:
This should be removed an internally replaced by 1e-12.

Explicit tolerance handling is addressed in #116.